covid_graphs/README.md

```diff
@@ -16,8 +16,9 @@ environment, install the `covid_graphs` package an run the server.
 python3.7 -m venv your/path/to/venv
 source your/path/to/venv/bin/activate
 pip install -e . # With -e the package will automatically reload with any local changes.
-server.py data build/results-poly.pb build/results-exp.pb
+covid_graphs.run_server --data-dir data/ --simulated-polynomial data/results-poly.pb --simulated-exponential data/results-exp.pb
```

Comment:
Not related to this review: I thought about this line and we have it somewhat inconsistent.

- We give it a directory, which contains data for 7 countries. It tries to intelligently pick all country files and skips those that aren't present.
- Then we give it as arguments two files in the same data directory, but with the full path, the app does no guessing and no skipping.

I think eventually we should drop the `--simulated-polynomial` and `--simulated-exponential` arguments and just use `--data-dir`.